src/components/Signup/Signup.js

```diff
@@ -8,102 +8,177 @@ import {
   Link,
   useHistory,
 } from "react-router-dom";
-import './Signup.scss' ;
+import "./Signup.scss";
 
 export default function Signup() {
   const History = useHistory();
   const routeToSignin = () => {
     History.push("/");
-
   };
 
+  const [formData, setformdata] = useState({
+    firstName: "",
+    lastName: "",
+    empCode: 15,
+    email: "",
+    password: "",
+    phone: "",
+    roleId: 1,
+    addressId: null,
+  });
+
+  function submitHandler() {
+    console.log("axios", formData);
+    axios
+      .post("https://localhost:44371/api/v1/user", formData)
+      .then((response) => console.log(response));
+  }
+  function clicker(e) {
```

Comment:
Can we have action explanatory function name like onChangeHandler

Reply:
changes done